docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+services:
+
+  mysqldb:
+    container_name: mysqldb
+    image: mysql:latest
+    environment:
+      - MYSQL_ROOT_PASSWORD=<password here>
+      - MYSQL_DATABASE=icu-connect
+      # - MYSQL_USER=root
+      # - MYSQL_PASSWORD=<password here>
+    restart: always
+    ports:
+      - 3307:3306
+    volumes:
+      - mysqldb:/var/lib/mysql
+
+  api:
+    container_name: api
+    image: icu-connect
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    environment:
+      SPRING_DATASOURCE_PASSWORD: <password here>
+      SPRING_DATASOURCE_USER: root
+      SPRING_DATASOURCE_NAME: icu-connect
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysqldb:3306/icu-connect?createDatabaseIfNotExist=true&autoReconnect=true
+    depends_on:
+      - mysqldb
+    ports:
+      - 8080:8080
+    restart: always
+
+volumes:
+  mysqldb:
```

Comment:
Should this contain a value?

Reply:
I don't think so, from what I gather this is standard procedure unless the volume is external to the docker-compose file as shown here https://docs.docker.com/storage/volumes/
![image](https://user-images.githubusercontent.com/55449626/225144135-e3d988b5-4afb-44c4-9a6f-cb2ce6b317ba.png)

Reply:
Great, thanks!